fedimintd/src/bin/distributedgen.rs

```diff
@@ -162,16 +162,13 @@ async fn main() -> anyhow::Result<()> {
                 &password,
                 configure_modules(max_denomination, network, finality_delay),
             )?;
-            let server =
-                match ServerConfig::distributed_gen(&params, module_registry(), &mut task_group)
-                    .await
-                {
-                    Ok(server) => server,
-                    Err(DkgError::Cancelled(_)) => return Ok(info!("DKG cancelled")),
-                    Err(DkgError::Failed(err)) => return Err(err),
-                };
-
-            write_server_config(&server, dir_out_path, &password, &module_registry())
+            let server = match ServerConfig::distributed_gen(&params, &mut task_group).await {
+                Ok(server) => server,
+                Err(DkgError::Cancelled(_)) => return Ok(info!("DKG cancelled")),
```

Comment:
LOL. Putting that `info!(...)` in there feels like cheating. :D

Reply:
Haha copying a pattern from elsewhere in the code, not sure who did it, might have been me.